graphics/opencv4/Portfile

```diff
@@ -190,26 +216,30 @@ variant nonfree description {Include nonfree algorithms} {
                         -DOPENCV_ENABLE_NONFREE:BOOL=ON
 }
 
-#variant contrib description {Build OpenCV with extra modules (untested)} {
-#    master_sites-append https://github.com/opencv/opencv_contrib/archive/:opencv_contrib
-#
-#    distfiles-append    ${distname}.zip:opencv_contrib
-#
-#    checksums-append    ${distname}.zip \
-#                        rmd160  a0dac9a5fc1c95d056a7523a6f37b8d1f600dd2b \
-#                        sha256  b4013495ac6c4dd05dcad1c90b6c731b488a1d775835175327f3c20884269715 \
-#                        size    61741612
-#
-#    depends_extract-append \
-#                        port:unzip
-#
-#    post-extract {
-#        system "${prefix}/bin/unzip -o '${distpath}/${distname}.zip' -d '${worksrcpath}'"
-#
-#        # less than ideal way for patchfiles to be applied to contrib directory
-#        ln -s ${workpath}/opencv_contrib-${version} ${worksrcpath}/opencv_contrib
-#    }
-#}
+variant opencv_contrib description {Build OpenCV with extra modules (untested)} {
```

Comment:
renaming the ```variant``` seem not needed here and will complicate upgrading for people who have the ```contrib``` variant already installed; please don't do it.

Reply:
This code was commented out before so the variant doesn't exist in prior versions.

Also, the GitHub project name is `opencv_contrib`, so it avoids confusion to use this as the variant name too.

Reply:
ah, okay - I missed that it was commented out before